spec/features/user_edits_their_profile_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+# [] Visiting the `/landmarks/1` path should show the landmark details for a landmark with the ID of 1.
+
+feature "user edits their profile" do
+  scenario "successfully edits profile" do
+    user = FactoryGirl.create(:user)
+
+    profile = Profile.create(
```

Comment:
we're having trouble relating these two classes if we use a factory for both, which is why we are manually passing in user for Profile